tests/util/skvbc_history_tracker.py

```diff
@@ -724,3 +729,110 @@ def _get_matching_request(self, rpy):
         causal_state = self.outstanding[(rpy.client_id, rpy.seq_num)]
         index = causal_state.req_index
         return (self.history[index], index)
+
+    async def verify_linearizability(self,test_case):
```

Comment:
I don't like the fact that we pass the test_case to the tracker. It shouldn't be aware of any test-specific stuff. Its job is to track and verify linearizability.

I suggest the following changes:
- create a new tracker instance for each test
- pass a bft_network instance to the tracker instance and use it internally where necessary (for creating a BFT client f.e.)
- refactor the debug statements (print ...), so that they don't depend on the specific test_case object